doc/user/content/sql/create-source/load-generator.md

```diff
@@ -30,6 +30,8 @@ _src_name_  | The name for the source.
 **AUCTION** | Use the [auction](#auction) load generator.
 **IF NOT EXISTS**  | Do nothing (except issuing a notice) if a source with the same name already exists.
 **TICK INTERVAL**  | The interval at which the next datum should be emitted. Defaults to one second.
+**FOR ALL TABLES** | Creates subsources for all data sources in the load generator.
+**FOR TABLES** _table_name_ | Creates subsources for specific data sources in the load generator.
```

Comment:
Should be added for PostgreSQL too right?

Reply:
Yeah!